frontend/src/components/markdown-renderer/extensions/emoji/emoji-markdown-extension.test.tsx

```diff
@@ -24,10 +24,8 @@ describe('Emoji Markdown Extension', () => {
     expect(view.container).toMatchSnapshot()
   })
 
-  it('renders a fork awesome code', () => {
-    const view = render(
-      <TestMarkdownRenderer extensions={[new EmojiMarkdownExtension()]} content={':fa-circle-thin:'} />
-    )
+  it('renders an icon code', () => {
```

Comment:
This is not rendered. Either the test has the wrong name or the test does not work